goosebit/api/v1/software/requests.py

```diff
@@ -1,5 +1,5 @@
-from pydantic import RootModel
+from pydantic import BaseModel
 
 
-class SoftwareDeleteRequest(RootModel[list[int]]):
-    pass
+class SoftwareDeleteRequest(BaseModel):
+    files: list[int]
```

Comment:
Naming is misleading: it is not really a list of files, but rather a list of software ids.